src/gmp-test/ChainBTokenSwapper.sol

```diff
@@ -4,31 +4,31 @@ pragma solidity 0.8.9;
 
 import { IERC20 } from '../interfaces/IERC20.sol';
 
-contract TokenSwapper {
+contract ChainBTokenSwapper {
     error WrongTokenPair();
 
-    address tokenA;
-    address tokenB;
+    address tokenX;
+    address tokenY;
 
-    constructor(address tokenA_, address tokenB_) {
-        tokenA = tokenA_;
-        tokenB = tokenB_;
+    constructor(address tokenX_, address tokenY_) {
+        tokenX = tokenX_;
+        tokenY = tokenY_;
```

Comment:
If x and y correspond to input and output respectively, then they should be named as such.

Reply:
Renaming back to A and B :)